kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/KafkaProxyReconciler.java

```diff
@@ -135,7 +138,6 @@ public ErrorStatusUpdateControl<KafkaProxy> updateErrorStatus(KafkaProxy primary
     }
 
     private static KafkaProxy buildStatus(KafkaProxy primary,
```

Comment:
I didn't rework the status building in the KPReconciler, but I think we should make it more consistent with what we're doing with the other reconcilers. But that's for another PR.